terrautils/sensors.py

```diff
@@ -118,75 +121,94 @@
               "fixed_metadata_datasetid": "TBD"
         },
 
-        'rgb_fullfield': {
+        'fullfield': {
+            'display': 'Full Field Stitched Mosaics',
```

Comment:
the plan had been to have the structure be like:
```
Full Field Stitched Mosaics
--- 2017
------ 2017-07
--------- 2017-07-23
------------ stereoTop_fullfield.tif
------------ stereoTop_fullfield_thumbnail.tif
------------ flirIr_fullfield.tif
------------ flirIr_fullfield_thumbnail.tif
...
```
And have the full field images together. We can split these into separate collections if you prefer.